repo_health/index/static/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "repo_health_frontend",
+  "version": "1.0.0",
+  "description": "Angular frontend for the repo health project.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
There are no dependencies for angular/ui router, so does the the index.html actually work?